src/components/Dashboard/style.css

```diff
@@ -0,0 +1,243 @@
+  border: 2px solid white;
+}
+
+.light:hover {
+  transform: translateY(-3px);
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.15);
+  border-color: #0099ff;
+}
+
+.light:active {
+  transform: translateY(0);
+  box-shadow: 0 3px 8px rgba(0, 0, 0, 0.15);
+  border-color: #0099ff;
+}
+
+.light__icon {
+  margin-right: 20px;
+}
+
+.light__icon img {
+  height: 45px;
+}
+
+.light__name {
+  flex: 1;
+  font-weight: bold;
+}
+
+/* klimatizace */
+.climate {
+  grid-area: climate;
+  display: flex;
+  align-items: center;
+  padding: 10px 20px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0 3px 8px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+  transition: border-color 0.2s;
+  border: 2px solid white;
+}
+
+.climate:hover {
+  border-color: #0099ff;
+}
+/* 
+.climate__icon {
+  /* margin-right: 20px 
+}*/
+
+.climate__icon img {
+  width: 40px;
+}
+
+.climate__content {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.climate__temperature {
+  margin-bottom: 10px;
+  font-size: 48px;
+  line-height: 1;
+  font-weight: 600;
+  text-align: center;
+}
+
+.climate__controls {
+  display: flex;
+  flex-direction: column;
+}
+
+.climate__controls button:first-child {
+  border-radius: 5px 5px 0 0;
+  border-bottom: 1px solid #bbbbbb;
+}
+
+.climate__controls button:last-child {
+  border-radius: 0 0 5px 5px;
+}
+
+/* žaluzie */
+.blinds {
+  grid-area: blinds;
+  display: flex;
+  align-items: center;
+  padding: 10px 20px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0 3px 8px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+  transition: border-color 0.2s;
+  border: 2px solid white;
+}
+
+.blinds:hover {
+  border-color: #0099ff;
+}
+
+.blinds__icon {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.blinds__icon img {
+  height: 40px;
+}
+
+.blinds__name {
+  flex: 1;
+  padding: 10px 20px 10px 0;
+  font-weight: bold;
+}
+
+.blinds__controls {
+  display: flex;
+}
+
+.blinds__controls button:first-child {
+  border-radius: 5px 0 0 5px;
+}
+
+.blinds__controls button:last-child {
+  border-radius: 0 5px 5px 0;
+}
+
+/* spotřeba energií */
+.energy {
+  grid-area: energy;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  padding: 10px 20px;
+  background-color: white;
+  border-radius: 5px;
+  box-shadow: 0 3px 8px rgba(0, 0, 0, 0.15);
+  cursor: pointer;
+  transition: border-color 0.2s;
+  border: 2px solid white;
+}
+
+.energy:hover {
+  border-color: #0099ff;
+}
+
+.energy__source {
+  display: flex;
+}
+
+.energy__icon {
+  margin-right: 15px;
+  display: flex;
+  align-items: center;
+}
+
+.energy__icon img {
+  display: block;
+  width: 25px;
+}
+
+.energy-consumption {
+  flex: 1;
+}
+
+.energy__description {
+  font-size: 14px;
+  font-weight: 300;
+  color: #666;
+}
+
+.energy__value {
+  font-size: 18px;
+  font-weight: 600;
+  color: #000;
+}
```

Comment:
tohle se ti opakuje třeba i v Energy/style.css, technicky vzato to chceš mít v aplikaci jen jednou. Ale je to pro teĎ fakt drobnost a hlavně jsme si ti víc neříkali o stylování, tak jen pro zajímavost 👍